FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/AvatarViewActivity.kt

```diff
@@ -62,6 +62,7 @@ class AvatarViewActivity : DemoActivity() {
         avatarView.avatarSize = AvatarSize.XXLARGE
         avatarView.name = getString(R.string.persona_name_mauricio_august)
```

Comment:
is it any more efficient to store the name in a local var and re-use for both cases? (and in cases below)

Reply:
Yea, it will be a bit faster. Incorporating the change